### ISSUE TITLE
[Figma Refactoring][Badge] Colors & Fonts not aligned

### ISSUE THREAD

**Issue by Gagne87** [baloise-incubator/design-system#1147]
I realised that we have following differences: 

- Colors don't match (much brighter in Storybook)

There are major accessibility issues at the moment with the colors used on Storybook (white font on warning-3 background e.g. only has a contrast ratio of 1.43:1 and fails). In this case I strongly recommend to adapt the colors according to our Figma. 

![Image](https://user-images.githubusercontent.com/29020214/219398947-53b4927f-e62a-4185-96ce-cf4155b6e9ef.png)

**Comment by hirsch88** [comment]
![Image](https://user-images.githubusercontent.com/6384499/221788783-fbd6f6d9-60a2-4cd9-b379-5f1ee445ee38.png)
